boxcli/root.go

```diff
@@ -41,6 +43,14 @@ func RootCmd() *cobra.Command {
 	command.AddCommand(RemoveCmd())
 	command.AddCommand(ShellCmd())
 	command.AddCommand(VersionCmd())
+
+	command.PersistentFlags().BoolVar(
```

Comment:
Should we mark this flag as hidden? It tends to pollute help messages because as a persistent flag it'll show up at the end of every command, and most users won't know what to do with the stack trace, unless we're asking for it (at which point we can tell them to re-run with `--debug`)

Reply:
Can we actually make this an environment variable? Something like `DEVBOX_DEBUG`? That's what Terraform does, and it's been really convenient since you don't need to remember to pass `--debug` all the time if you're a developer. You just set it once in your shell profile and forget about it.